integration-cli/docker_cli_plugins_test.go

```diff
@@ -3,14 +3,18 @@ package main
 import (
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/go-check/check"
+
+	"strings"
+)
+
+var (
+	name        = "tiborvass/no-remove"
```

Comment:
Fixed. Eventually, I plan to have a utils/helper file for plugins related tests. That file will have these globals.